docs/pages/build-reference/variables.mdx

```diff
@@ -73,6 +73,8 @@ To provide your build jobs with access to values that are too sensitive to inclu
 
 A secret is made up of a name and a value. The name can only contain alphanumeric characters and underscores. The value is limited to 32 KiB.
 
+The value can be either a file or a string value. For a file, it's restored on EAS Build servers at a temporary file path. The path is available via the environment variable. For example, if you created a file secret named `SECRET_FILE`, EAS Build will create a file at `/Users/expo/workingdir/environment-secrets/__UNIQUE_RANDOM_UUID__`, and `SECRET_FILE` will be set to that path.
```

Comment:
```suggestion
The value can be either a file or a string value. For a file, it's created on EAS Build servers at a temporary file path. The path is available via the environment variable. For example, if you created a file secret named `SECRET_FILE`, EAS Build will create a file at `/Users/expo/workingdir/environment-secrets/__UNIQUE_RANDOM_UUID__`, and `SECRET_FILE` will be set to that path.
```

Reply:
How strong is your opinion on this?

Reply:
Well, nothing is restored, so pretty strong.

Reply:
Technically, it is restored :D The file existed on the user's computer (not at the same path but).